core/domain/collection_services.py

```diff
@@ -904,9 +1128,13 @@ def get_next_page_of_all_commits(
         page_size=feconf.COMMIT_LIST_PAGE_SIZE, urlsafe_start_cursor=None):
     """Returns a page of commits to all collections in reverse time order.
 
-    The return value is a triple (results, cursor, more) as described in
-    fetch_page() at:
+    Args:
+        page_size: int. Size of the page of commits to be returned.
+        urlsafe_start_cursor: Cursor pointing to the first page of commits.
```

Comment:
str or None. If provided, the list of returned commits starts from this datastore cursor. Otherwise, the returned commits start from the beginning of the full list of commits.

(Btw, please check to ensure that None cases are documented adequately. It looks like they've been left out in some docstrings.)